packages/webapp/src/components/Typography/index.tsx

```diff
@@ -31,6 +31,7 @@ export const Underlined = ({
 type IconLinkProps = TypographyProps & {
   icon?: ReactNode;
   isIconClickable?: boolean;
+  underlined?: boolean;
```

Comment:
I noticed there are a few places where we use these type of icon link where the text is grey and not underlined, so to avoid having to use CSS specificity to override the underlined styling every time, I figured I'd just add a prop here to make that more flexible.